lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -282,7 +291,8 @@ newStakePoolLayer nl db@DBLayer {..} worker = do
         case r of
             Left e@(PastHorizon{}) ->
                 throwE (ErrListPoolsPastHorizonException e)
-            Right r' -> pure r'
+            Right r' -> pure (ApiListStakePools r'
+                (Just . ApiT . Iso8601Time . posixSecondsToUTCTime $ lastGC))
```

Comment:
**Suggestion:** Simplify the mixture of `$`, `.`, and parentheses.

```suggestion
            Right r' -> pure  
                $ ApiListStakePools r'
                $ Just
                $ ApiT
                $ Iso8601Time 
                $ posixSecondsToUTCTime lastGC
```

Reply:
This is many lines longer.

Reply:
> This is many lines longer.

Fair enough! As an alternative suggestion:

```hs
            Right r' -> pure  
                $ ApiListStakePools r'
                $ Just $ ApiT
                $ Iso8601Time $ posixSecondsToUTCTime lastGC
```